### PR TITLE
Issue #93: change references to EOT Harvester in the seeder documentation

### PR DESCRIPTION
**Pull request by janetriley**
Changed references to End of Term to Internet Archive and updated URL.

### CODE REVIEW

**Comment by janetriley** [comment]
Ping for review:  @khdelphine @dcwalk @liblaurie @titaniumbones

File: docs/seeding.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## What Do Seeders Do?
 
-Seeders canvass the resources of a given government agency, identifying important URLs and whether those URLs can be crawled by the Internet Archive's web crawler. They use the [EDGI Nomination Chrome extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en) to nominate URLs to the [End of Term (EOT) Web Archive](http://eotarchive.cdlib.org/2016.html) if they are crawlable or to the Archivers app if they require manual archiving.
+Seeders canvass the resources of a given government agency, identifying important URLs and whether those URLs can be crawled by the Internet Archive's web crawler. They use the [EDGI Nomination Chrome extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en) to nominate URLs to the [Internet Archive](http://www.archive.org) if they are crawlable or to the Archivers app if they require manual archiving.
```
Comment (dcwalk) [inline docs/seeding.md:3]: I think we might need to update the "nominate" language as well, something like "save URLS to the Internet Archive" (as that is the terminology IA uses).

File: docs/seeding.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Seeders use the [EDGI Archiving Primers](https://envirodatagov.org/archiving/),
 
 ### Crawlable URLs
 
-- URLs judged to be crawlable are "nominated" (equivalently, "seeded") to the End of Term project (EOT), using the [EDGI Nomination Chrome extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en).
+- URLs judged to be crawlable are "nominated" (equivalently, "seeded") to the Internet Archive, using the [EDGI Nomination Chrome extension](https://chrome.google.com/webstore/detail/nominationtool/abjpihafglmijnkkoppbookfkkanklok?hl=en).
```
Comment (dcwalk) [inline docs/seeding.md]: Awesome, again tho to be consistent with language I think we want something like "save URLS to the Internet Archive"

**Review by dcwalk — COMMENTED** [review]
Thanks for the quick work, I suggested a language change

**Comment by dcwalk** [comment]
Also--question... do we want to add an explainer about the change to "saving" vs. "nominating"? (maybe as a note, see below)
```
<div class = "attention">
  <strong>Note: [Name]</strong> <br />  
  [Text]
</div>
```

**Comment by khdelphine** [comment]
The edits look great.

@dcwalk, to me the word "save" seems to imply that the action is instantaneous, which is not the case. Or am I overthinking this?

**Comment by dcwalk** [comment]
@khdelphine -- we are moving to implement a direct save using the IA's API now that the End of Term crawl is over.

**Comment by khdelphine** [comment]
@dcwalk Ah ok, I had missed that. Perfect!

**Comment by dcwalk** [comment]
Once the changes are made, I think we want to hold off/ensure we merge this in at the right time (i.e., in sync with the App update)

File: docs/seeding.md (hunk 1 above)
Comment (janetriley) [reply docs/seeding.md:3]: Done.

**Comment by janetriley** [comment]
The language is updated. 

Re: Explainer note about the change - is anyone outside the project going to notice?  It strikes me as one of those changes that's significant when you make the update but you forget two months later.

**Review by dcwalk — APPROVED** [review]

**Comment by dcwalk** [comment]
Fair point @janetriley. This looks great! I'm just going to pause a bit on this to ensure we are in sync with the convo in the Chrome Extension repo: https://github.com/edgi-govdata-archiving/eot-nomination-tool/issues/72

Ping me by Friday if I haven't moved on this!

**Comment by dcwalk** [comment]
Sorry for the delay on this, it took awhile to pin down our process with IA. We are gonna keep using the nominate language, merging in for the weekend :)

**Pull request merged**